app/Http/Controllers/StudentsCouncil/AnonymousQuestionController.php

```diff
@@ -193,4 +143,17 @@ public function exportAnswerSheets(Semester $semester)
             'anonymous_questions_' . $semester->year . '_' . $semester->part . '.xlsx'
         );
     }
+
+    public function delete(Semester $semester, Question $question)
+    {
+        $this->authorize('administer', AnswerSheet::class);
+
+        if($question['parent_type'] != AnswerSheet::class) {
+            abort(400);
+        }
```

Comment:
probably irrrelevant with `scopeBindings()` as that ensurers the question is the child of the specified general assembly.